src/Microsoft.OData.Core/UriParser/Binders/SelectExpandBinder.cs

```diff
@@ -524,7 +548,7 @@ private ComputeClause BindCompute(ComputeToken computeToken, IEdmNavigationSourc
         /// <param name="filterToken">The filter token to visit.</param>
         /// <param name="resourcePathNavigationSource">The navigation source at the resource path.</param>
         /// <param name="targetNavigationSource">The target navigation source at the current level.</param>
-        /// <param name="elementType">The Edm element type.</param>
+        /// <param name="elementType">The target element type if different from the type of the target navigation source.</param>

```

Comment:
I think it would be really good for this updated documentation to specify that `null` indicates that the element type is the same as the target navigation source type

Reply:
#Resolved